packages/js-client-rest/src/api/snapshots-api.ts

```diff
@@ -47,6 +47,14 @@ export function createSnapshotsApi(client: Client) {
          */
         listSnapshots: client.path('/collections/{collection_name}/snapshots').method('get').create(),
 
+        /**
+         * Recover local collection data from an uploaded snapshot. This will overwrite any data, stored on this node, for the collection. If collection does not exist - it will be created.
+         */
+        recoverFromUploadedSnapshot: client
```

Comment:
We don't use this endpoint in client, but I added it for consistency (like `recoverShardFromUploadedSnapshot`)